Cargo.toml

```diff
@@ -18,6 +18,15 @@ tracing-subscriber = { version = "0.3", default-features = false, features = [
   "fmt",
   "json",
 ] }
+serde = { version = "1.0.163", optional = true }
+serde_json = { version = "1.0.96", optional = true }
+opentelemetry_api = { version = "0.19.0", features = ["testing"], optional = true }
+rand = { version = "0.8.5", optional = true }
+
+[features]
+full = []
+default = ["integration_test"]
```

Comment:
Is this correct? Shouldn't we disable `integration_test` feature by default?

Reply:
Perhaps you meant

```rust
full = ["integration_test"]
default = []
```

Reply:
Perhaps you meant

```rust
full = ["integration_test"]
default = []
```

Reply:
oops, I flipped it in the last commit